test/library/packages/Python/examples/parquet/readParquet.cleanfiles

```diff
@@ -0,0 +1,2 @@
+data.parquet
+readParquet.good
```

Comment:
Since we're generating the .good file, it should be added to a .gitignore as well, right?

Reply:
Yup there should be a .gitignore. Looks like I left it behind in my buffer, pushed now